install_config/configuring_sdn.adoc

```diff
@@ -348,6 +348,18 @@ $ oc get netnamespace
 [[migrating-between-sdn-plugins-networkpolicy]]
 === Migrating from ovs-multitenant to ovs-networkpolicy
 
+[NOTE]
+====
+Only the `v1` NetworkPolicy features are available in {product-title}. This
```

Comment:
Lately I'd say it this way:

"The `v1` NetworkPolicy features are available only in {product-title}."

But I suffer from poor placement of _only_, so I'd get confirmation.

Reply:
+1

Reply:
Fixed! Thank you!